lib/util/cache/memory/index.ts

```diff
@@ -1,19 +1,29 @@
-let repoCache: Record<string, any> | undefined;
+let memCache: Record<string, any> | undefined;
 
 export function init(): void {
-  repoCache = {};
+  memCache = {};
 }
 
 export function reset(): void {
-  repoCache = undefined;
+  memCache = undefined;
 }
 
 export function get<T = any>(key: string): T {
-  return repoCache?.[key];
+  return memCache?.[key];
 }
 
 export function set(key: string, value: unknown): void {
-  if (repoCache) {
-    repoCache[key] = value;
+  if (memCache) {
+    memCache[key] = value;
+  }
+}
+
+export function cleanDatasourceKeys(): void {
+  if (memCache) {
+    for (const key of Object.keys(memCache)) {
+      if (key.startsWith('global%%') || key.startsWith('datasource-releases')) {
```

Comment:
The first one caches promises of the `getReleases()`, the second one caches the result of this promise (enabled by `caching=true` static field)

Reply:
Can we rename them so it's more intuitive?